New version: Redis v2.1.0

JuliaRegistrator opened the pull request:
- Registering package: Redis
- Repository: https://github.com/JuliaDatabases/Redis.jl
- Created by: @tanmaykm
- Version: v2.1.0
- Commit: 13d925a08c92834d49a72925b72152ef70f3769f
- Reviewed by: @tanmaykm
- Reference: https://github.com/JuliaDatabases/Redis.jl/commit/13d925a08c92834d49a72925b72152ef70f3769f#commitcomment-136102490
- Description: A fully-featured Redis client for the Julia programming language
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd68b19e5a79c2e01b2f88b987b277f7ddd82bcf4e8cce308af56028e4ca1d1fe3853f55e8467dbf1c3b1c79315d3db28e6f2c8e76d5d9533bb78a4a7c222604dd65b5b0d648872ac919a7021b9d0e712e3c43a142853b083eeae388eb7f6abe53368168eecbf992067d4687fe0144318ed7fa3b8910029585d195d2f306b7eab21b725d6bbec69913932c620b8530d0c4c79e97b8fd6c9a5ac9e6207dd9bdbc01901a83dcd9c0df47f5443f05415b51f65 -->

tanmaykm commented:
Looks like this [earlier PR to tag v2.0.0](https://github.com/JuliaRegistries/General/pull/94851) had failed and was ignored. The [github repo releases](https://github.com/JuliaDatabases/Redis.jl/releases) were probably created manually and not by tagbot.

It is probably fine to ignore the old release and go ahead with v2.1.0.
cc: @jkaye2012

JuliaRegistrator committed:
New version: Redis v2.1.0

UUID: 0cf705f9-a9e2-50d1-a699-2b372a39b750
Repo: https://github.com/JuliaDatabases/Redis.jl.git
Tree: 73dc3016ea6acba2dc7e1fdcb50fc0233f56c335

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

tanmaykm commented:
Can we go ahead with a manual merge for this?
Maybe @giordano or @DilumAluthge can help?
